### PR TITLE
[master]Add restart action after catpure active VM completed successfully.

### PR DESCRIPTION
**Pull request by wngzhe**
code change:
1. Add VM start operation after capture active VM completed successful.
2. change unit test cases.

### CODE REVIEW

File: zvmsdk/smtclient.py [hunk 1]
```diff
@@ -1087,6 +1087,8 @@ def guest_capture(self, userid, image_name, capture_type='rootonly',
 
             # Shutdown the vm before capture
             self.guest_softstop(userid)
+            # keep restart flag used after capture.
+            restart_flag = True
```
Comment (jichenjc) [inline zvmsdk/smtclient.py:1091]: what about a paused VM?

File: zvmsdk/smtclient.py [hunk 2]
```diff
@@ -1087,6 +1087,8 @@ def guest_capture(self, userid, image_name, capture_type='rootonly',
 
             # Shutdown the vm before capture
             self.guest_softstop(userid)
+            # keep restart flag used after capture.
```
Comment (jichenjc) [inline zvmsdk/smtclient.py:1090]: should we init the restart flag at beginning ? or we are sure this flag will be inited anyway?

File: zvmsdk/smtclient.py (hunk 1 above)
Comment (wngzhe) [reply zvmsdk/smtclient.py:1091]: I will get status restore function to make sure status keep same,,,

File: zvmsdk/smtclient.py (hunk 2 above)
Comment (wngzhe) [reply zvmsdk/smtclient.py:1090]: we have 2 branches for get_guest_connection_status, both branch has set flags,  
flag will be inited any way,

File: zvmsdk/smtclient.py (hunk 1 above)
Comment (wngzhe) [reply zvmsdk/smtclient.py:1091]: so far we can't get  pause vm status precisly , we just check  power status ,all VM with power status  on  will be  stop before capture and start after capture ,  power off status VM will not start, 
bu this we can make active vm and shutoff vm status same before capture as after capture,
but for pause vm , after capture vm ,the VM will be start.

File: zvmsdk/smtclient.py [hunk 3]
```diff
@@ -1120,6 +1123,13 @@ def guest_capture(self, userid, image_name, capture_type='rootonly',
                 LOG.error(msg)
                 raise exception.SDKGuestOperationError(rs=5, userid=userid,
                                                        msg=msg)
+            # if VM power on, the VM need be perform stop and start
+            power_state = self.get_power_state(userid)
+            if power_state == 'on':
+                # Shutdown the vm before capture
+                self.guest_softstop(userid)
```
Comment (bjhuangr) [inline zvmsdk/smtclient.py:1130]: if not reachable but power state on, then softstop will not work

**Review by bjhuangr — CHANGES_REQUESTED** [review]

File: zvmsdk/smtclient.py (hunk 3 above)
Comment (SeanHQF) [reply zvmsdk/smtclient.py:1130]: @wngzhe Then a reachable check is needed?

Comment (wngzhe) [reply zvmsdk/smtclient.py]: will change to guest_stop

**Review by bjhuangr — APPROVED** [review]
LGTM

**Pull request merged**